api/v1/app.py

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/python3
+"""App module
+"""
+from flask import Flask
+from models import storage
+from api.v1.views import app_views
+from os import getenv
+
+
+app = Flask(__name__)
+app.config['JSONIFY_PRETTYPRINT_REGULAR'] = True
+app.register_blueprint(app_views, url_prefix='/api/v1')
+
+
+@app.teardown_appcontext
+def teardown_db(error):
+    """Closes the database again at the end of the request.
+    """
+    storage.close()
+
+
+if __name__ == "__main__":
+    host = getenv('HBNB_API_HOST', '0.0.0.0')
+    port = getenv('HBNB_API_PORT', 5000)
```

Comment:
For improving code legibility typing default=value on get env functions "could be better"

Reply:
Of course!